### DISCUSSION TITLE
Debug "Failed to Import" toast message

### DISCUSSION THREAD

**Discussion by chantlong** [directus/directus#16853] [Archive]
When I import a CSV into a Directus Table through the GUI, I sometimes get the "Failed to Import" message instantly, sometimes after 20 seconds of loading.
When the toast is instant, I see that my record count doesn't change. 

The ones with 20 seconds of loading actually does show my record count goes up.
So despite it saying "Failed to import" in the end, it does seem like everything is getting imported at first glance.

Is there anyway I can debug what the real error message is for "Failed to Import"?
I need to make sure everything is imported correctly.

**Comment by paescuj** [comment]
Did you check the server console for errors? Would you be able to provide a CSV file which is causing this issue?

**Reply by chantlong** [reply]
So it turns out because it takes forever, I end up running into a 504 Gateway Timeout. That was the issue.
So I had to break my import into chunks.
